app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomFragment.kt

```diff
@@ -339,6 +339,14 @@ class ChatRoomFragment : Fragment(), ChatRoomView, EmojiKeyboardListener, EmojiR
             }
 
             if (recycler_view.adapter == null) {
+                adapter = ChatRoomAdapter(
```

Comment:
The adapter is already initialized on `onCreate` don't create it twice. Also, this is not compiling...